src/schemas/components/SchemaFormPropertyArray.vue

```diff
@@ -45,92 +21,48 @@
     state: State,
   }>()
 
+  const schema = useSchema()
   const { property } = useSchemaProperty(() => props.property)
-  const empty = computed(() => !props.value?.length)
 
   const emit = defineEmits<{
     'update:value': [unknown[] | undefined],
   }>()
 
-  const value = computed({
-    get() {
-      return props.value ?? []
-    },
-    set(value) {
-      if (value.length === 0) {
-        emit('update:value', undefined)
-        return
-      }
-
-      emit('update:value', value)
-    },
-  })
-
-  const allowCreate = computed(() => {
-    const max = props.property.maxItems ?? Infinity
-    const current = props.value?.length ?? 0
-
-    return current < max
-  })
-
-  const showMaxItemsMessage = computed(() => {
-    const max = props.property.maxItems ?? Infinity
-    const current = props.value?.length ?? 0
-
-    return current >= max
-  })
-
-  function getPropertyForIndex(index: number): SchemaProperty {
-    if (isArray(property.value.prefixItems)) {
-      return property.value.prefixItems[index] ?? {}
-    }
-
-    if (isArray(property.value.items)) {
-      return property.value.items[index] ?? {}
-    }
-
-    return property.value.items ?? {}
-  }
-
-  function generator(): unknown {
-    const index = value.value.length
-    const property = getPropertyForIndex(index)
-
-    return property.default ?? undefined
+  function isSelectModalValue(value: unknown): value is SelectModelValue {
+    return isNumber(value) || isString(value) || isBoolean(value) || isNull(value)
   }
 
-  function isFirstIndex(index: number): boolean {
-    return index === 0
-  }
+  function asSelectModalValue(value: unknown): SelectModelValue[] | undefined {
```

Comment:
```suggestion
  function asSelectModelValue(value: unknown): SelectModelValue[] | undefined {
```

Reply:
ty, fixed in https://github.com/PrefectHQ/prefect-ui-library/pull/2252/commits/c17012aa5c3e8215f0c1760703a832a5bf6c41f8